domains/transfer/Transfer.API/API/Program.cs

```diff
@@ -24,6 +30,28 @@
 
 builder.AddSerilog();
 
+builder.Services.AddHttpClient<IAuthorizationClient, AuthorizationClient>(client =>
+{
+    client.BaseAddress = new Uri(builder.Configuration["Authorization:BaseUrl"]!); // Do we want to fail in other way than nullpointer?
+})
+.AddPolicyHandler(RetryPolicy())
```

Comment:
Adding Retry and Circuit breaker policy. This could be a start of something new? https://www.youtube.com/watch?v=zBG-sEGlULs